packages/dds/merge-tree/src/test/revertibleFarm.spec.ts

```diff
@@ -205,6 +205,7 @@ describe("MergeTree.Client", () => {
 						errorPrefix: "After Re-Revert (redo)",
 						baseText: redoBaseText,
 					});
+					logger.dispose();
```

Comment:
noticed these tests were spamming max event listener attach events. leaving the loggers attached isn't a huge deal but it's a waste since they do nontrivial work in delta handling. this change improves suite time ~30s->~10s on my machine.

Reply:
can you add one to packages\dds\merge-tree\src\test\client.rollbackFarm.spec.ts as well. i think we missed these in the transition to events from callbacks